### PR TITLE
Preserves your current camera position when you stop following another user

### PR DESCRIPTION
**Pull request by benptc**
Also lets you click the following info button to stop following, in addition to pressing escape.

### CODE REVIEW

File: content_scripts/VirtualCamera.js [hunk 1]
```diff
@@ -1080,13 +1080,57 @@ import Splatting from '../../src/splatting/Splatting.js';
...
+            // Extract direction from the third row (indices 8, 9, 10), and negate
+            let direction = [
+                -cameraRelativeToGroundPlane[8],
+                -cameraRelativeToGroundPlane[9],
+                -cameraRelativeToGroundPlane[10]
+            ];
+
+            // Normalize the direction vector
+            const magnitude = Math.sqrt(direction[0] * direction[0] + direction[1] * direction[1] + direction[2] * direction[2]);
+            direction = direction.map(component => component / magnitude);
```
Comment (Steve-KX-RL) [inline content_scripts/VirtualCamera.js:1123]: Maybe using some functions below to make it more concise?
direction = normalize(direction);

File: content_scripts/VirtualCamera.js [hunk 2]
```diff
@@ -1080,13 +1080,57 @@ import Splatting from '../../src/splatting/Splatting.js';
...
+
+            // Calculate target position as position + direction (but move the focus a certain distance in front)
+            const targetPosition = [
+                position[0] + direction[0] * FOCUS_DISTANCE_MM_IN_FRONT_OF_VIRTUALIZER,
+                position[1] + direction[1] * FOCUS_DISTANCE_MM_IN_FRONT_OF_VIRTUALIZER,
+                position[2] + direction[2] * FOCUS_DISTANCE_MM_IN_FRONT_OF_VIRTUALIZER
+            ];
```
Comment (Steve-KX-RL) [inline content_scripts/VirtualCamera.js:1130]: Same comment as above:
const targetPosition = add(position, scalarMultiply(direction, FOCUS_DISTANCE_MM_IN_FRONT_OF_VIRTUALIZER));

**Review by Steve-KX-RL — CHANGES_REQUESTED** [review]
Just found another bug:
When a user is in lock on mode, if they still performs rotate / pan / zoom, after exiting lock on mode, when the user performs an action, the camera would drift really far away.
This is due to we didn't clear this.mouseInput.unprocessedXXX when in lock on mode.
We could modify it to:

            if (this.lockOnMode) {
                this.updateLockOnMode();
                this.mouseInput.unprocessedDX = 0;
                this.mouseInput.unprocessedDY = 0;
                this.mouseInput.unprocessedScroll = 0;
                return; // stop executing - cannot zoom or pan while in lock-on mode
            }

in the update() function.

File: content_scripts/VirtualCamera.js (hunk 2 above)
Comment (benptc) [reply content_scripts/VirtualCamera.js:1130]: Thanks for the comments and for noticing that bug! I'll fix them now!

**Review by Steve-KX-RL — APPROVED** [review]
LGTM!

**Pull request merged**